spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -122,6 +124,39 @@
         chartColorsPerHttpResponseCode = Collections.unmodifiableMap(colorMap);
     }
 
+    private static final int convertTimestampMilisecondsToMinutes = 60000;
```

Comment:
This is a functions name, since functions are 'doing things' like converting. I would name it to something like ```oneMinuteInMillis```.

Reply:
Done